service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClient.java

```diff
@@ -229,6 +261,11 @@ public void send(String deviceId, String moduleId, Message message) throws IOExc
      */
     @Deprecated public FeedbackReceiver getFeedbackReceiver(String deviceId)
     {
+        if (options != null)
```

Comment:
this now needs to be `options.getProxyOptions() != null`

Reply:
Good catch, you are right